quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -421,8 +421,13 @@ impl BBRv2 {
         BBRv2 {
             mode: Mode::startup(BBRv2NetworkModel::new(&params, initial_rtt)),
             cwnd,
-            pacing_rate: Bandwidth::from_bytes_and_time_delta(cwnd, initial_rtt) *
-                params.startup_pacing_gain,
+            // sRTT is not known yet.  Compute the initial pacing rate by
+            // dividing cwnd by 1msec as recommented by draft-ietf-ccwg-bbr-03.
+            // https://www.ietf.org/archive/id/draft-ietf-ccwg-bbr-03.html#name-pacing-rate-cpacing_rate
+            pacing_rate: Bandwidth::from_bytes_and_time_delta(
+                cwnd,
+                Duration::from_millis(1),
+            ) * params.startup_pacing_gain,
```

Comment:
From what I can tell this is changing behavior. We currently use [INITIAL_RTT](https://github.com/cloudflare/quiche/blob/master/quiche/src/lib.rs#L447). At the very least we might want to match the calculation in the RFC: `SRTT ? SRTT : 1ms` but if we are using INITIAL_RTT I don't expect this to every apply.

---

Also can you add the pseudo code as a comment here if we want this behavior.
```
  BBRInitPacingRate():
    nominal_bandwidth = C.InitialCwnd / (SRTT ? SRTT : 1ms)
    C.pacing_rate =  BBR.StartupPacingGain * nominal_bandwidth
```

Reply:
An initial RTT estimate of 333ms is not a valid sRTT measurement.  It is the absence of a measurement, so the code should really be using 1msec when computing the initial pacing rate.